SU-HSDO/suhumsci
# feat (STN-506): Colorful Carousel Height Refinement

# READY FOR REVIEW

## Summary
Refine the height of the carousel in the Colorful theme. This PR reduces the height of the carousel in Colorful by approximately 200px.

Reference: [JIRA STN-506](https://sparkbox.atlassian.net/browse/STN-506)

## Need Review By (Date)
8/28

## Urgency
medium

## Steps to Test
1. In the CLI, run 'npm run test' and confirm our tests pass.
2. To review the work in this PR, set the theme to Colorful.
3. Go to http://sparkbox-sandbox.suhumsci.loc/qa/carousel-hero-image to review the carousel
    - [x] Review  carousel as a hero at the top of the page
    - [x] Review carousel within the page content
4. Confirm all changes only effect the Colorful theme carousel. The Traditional theme carousel should remain the same. Note that there appears to be a regression on the Traditional Carousel. This is on the sprint branch.

### Top of page in hero area
#### Before
![original](https://user-images.githubusercontent.com/12678977/91474440-37491280-e868-11ea-9011-9ac340fe4a82.png)

#### After
![after](https://user-images.githubusercontent.com/12678977/91474865-d0782900-e868-11ea-9ff0-e3f66b384c1c.png)

### Within main content of page
#### Before
![carousel in content BEFORE](https://user-images.githubusercontent.com/12678977/91474575-665f8400-e868-11ea-8ab7-22bca7ec7a08.png)

#### After
![after in content](https://user-images.githubusercontent.com/12678977/91474913-dbcb5480-e868-11ea-9105-b46fc2883b2b.png)
